README.md

```diff
@@ -1 +1,77 @@
-# hello-microverse-project
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+# 📖 [Hello Microverse project]
+
+**[Hello Microverse project]** is a simple project that display a hello microverse message.
+
+## 🛠 Built With
+
+### Tech Stack
+
+**Client**
+React.js
+**Server**
+Express.js
+**Database**
+PostgreSQL
+
+### Key Features
+
+- **[When clicked it prints "Hello Microverse" in green color]**
+
+## 🚀 Live Demo
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
```

Comment:
I suggest you take away this link as it doesn't relate to the project, you can put `no live demo` or something else since this milestone doesn't require a live demo😉 putting a wrong link will make your project look less professional🤗🤗

Reply:
Done!
Check Update README file. pull